_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,79 @@
+- [Notebooks Dashboards Plugin](https://github.com/opensearch-project/dashboards-notebooks/blob/main/release-notes/opensearch-dashboards-notebooks.release-notes-1.0.0.0-rc1.md)
+- [Reporting Opensearch Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Reporting Dashboards Plugin](https://github.com/opensearch-project/dashboards-reports/blob/main/release-notes/opensearch-dashboards-reports.release-notes-1.0.0.0-rc1.md)
+- [Trace Analytics Dashboards Plugin](https://github.com/opensearch-project/trace-analytics/blob/main/release-notes/opensearch-trace-analytics.release-notes-1.0.0.0-rc1.md)
+- [Gantt Charts Dashboards Visualization Plugin](https://github.com/opensearch-project/dashboards-visualizations/blob/main/release-notes/opensearch-dashboards-visualizations.release-notes-1.0.0.0-rc1.md)
+- [Async-Search](https://github.com/opensearch-project/asynchronous-search/blob/main/release-notes/opensearch-asynchronous-search.release-notes-1.0.0.0-rc1.md)
+- [Common-Utils](https://github.com/opensearch-project/common-utils/blob/main/release-notes/opensearch-common-utils.release-notes-1.0.0.0-rc1.md)
+
+Consolidated release notes can be found [here](https://github.com/opensearch-project/OpenSearch/tree/main/release-notes). 
+
+**What does this Release Candidate not include?** 
+
+* *Minimum Artifacts* - Based on community [feedback](https://github.com/opensearch-project/opensearch-build/issues/31), we plan on releasing standalone, minimal downloadable artifacts without OpenSearch plugins (alerting, AD, security, etc.). We are expecting to deliver these artifacts by GA.
+* *Additional Artifacts* - In addition to the current artifacts (Linux tars and Docker images) we are aiming to deliver RPM (X64), Windows (X64), and DEB (X64) by GA and Tar (ARM64), DEB (ARM64), RPM (ARM64), MacOS (M1 ARM), and MacOS (X64) by OpenSearch 1.1. 
+* *Clients with License Checks* - We are aware of the issue with license checks which are causing incompatibilities (i.e. the recent of Logstash & Beats 7.13) and are working on a solution for the community before GA.
+
+**How can you help?** 
+
+The best thing you can do is run the Release Candidate through its paces. Every bit of testing and/or feedback the community can provide helps ensure that this Release Candidate is performing as expected and we are building towards the best possible GA release. Couple specific places to start;
+
+- Install the Release Candidate and open an [issue](https://github.com/opensearch-project/OpenSearch/issues) to report any bugs you discover. 
+- Run rolling upgrades from existing version to OpenSearch RC1 in a sandbox cluster. 
+- Test clients! We want to know what is working (and not working) for you. 
+- Use our [docs](https://docs-beta.opensearch.org/) and highlight any additional content you'd like to see! Or submit a PR and add any content you think would be valuable.
```

Comment:
You might want to be more more precise and inclusive here.
'submit a PR or Issue to the [docs]` link to docs githu

Reply:
Done, thanks!